### PR TITLE
[SM-664] Checking if the user has selected access tokens to revoke, if not err…

### PR DESCRIPTION
**Pull request by cd-bitwarden**
…or message

## Type of change

```
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Check if the user has selected access token to revoke, if not show an error message.

Additionally, make the `bwi-minus-circle` red instead of black to be consistent with the other dropdown colors.

## Code changes

- `apps/web/src/locales/en/messages.json` - adding localization for the phrase of "no access tokens selected to revoke"
- `bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/access/access-tokens.component.ts` - checking if the tokens exist that we are trying to delete before doing logic to revoke them
- `bitwarden_license/bit-web/src/app/secrets-manager/service-accounts/access/access-tokens.component.html` - make the `bwi-minus-circle` red

## Screenshots

The toast error message:

access-token

Make the `bwi-minus-circle` red instead of black. The old version:

Screenshot 2023-05-03 at 15 10 33

The new version:

revoke-token

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -6165,6 +6165,10 @@
     "message": "Access tokens revoked",
     "description": "Toast message after deleting one or multiple access tokens."
   },
+  "noAccessTokenSelected": {
+    "message": "No access token selected to revoke",
+    "description": "Toast message after trying to delete access tokens but not selecting and access tokens."
```
Comment (coltonhurst) [inline apps/web/src/locales/en/messages.json:6170]: Should this be `"Toast error message after trying to delete access tokens but not selecting any access tokens."`? (Main change was changing `and` to `any`)

**Review by coltonhurst — CHANGES_REQUESTED** [review]
Looks good, just one question about the description in `messages.json`.

**Review by coltonhurst — CHANGES_REQUESTED** [review]
Looks great! One other thought - to match other areas we could check `!tokens?.length` up front to reduce indentation in the `revoke` function, and to handle if tokens is ever `null` or `undefined`. Let me know what you think?

```
if (!tokens.length) {
  this.platformUtilsService.showToast(
    "error",
    null,
    this.i18nService.t("noAccessTokenSelected")
  );
  return;
}

if (!(await this.verifyUser())) {
  return;
}

await this.accessService.revokeAccessTokens(
  this.serviceAccountId,
  tokens.map((t) => t.id)
);

this.platformUtilsService.showToast("success", null, this.i18nService.t("accessTokenRevoked"));
```

**Review by Thomas-Avery — APPROVED** [review]
LGTM

**Pull request merged**